src/app/components/nodes-view/nodes-view.component.html

```diff
@@ -34,8 +34,7 @@
   <div class="right-wrapper">
     <div class="filter">
       <mat-form-field>
-        <mat-label>Filter</mat-label>
-        <input matInput (keyup)="applyFilter($event)" placeholder="Ex. amd64" #input>
+        <input matInput (keyup)="applyFilter($event)" placeholder="Filter (Ex. amd64)" #input>
```

Comment:
The example "(Ex. amd64)" is meaningless to me. Do we need it?

<img width="1253" alt="image" src="https://github.com/user-attachments/assets/a18204e9-6301-4423-951f-99752fb24e24">

Reply:
I thought the "amd64" doesn't provide any useful information here.